src/datasource.js

```diff
@@ -933,7 +933,9 @@ ngeo.DataSource.OGCType = {
  * @enum {string}
  */
 ngeo.DataSource.WFSFeatureNS = {
-  MAPSERVER: 'http://mapserver.gis.umn.edu/mapserver'
+  GEOSERVER: 'http://www.opengis.net/gml/3.2',
```

Comment:
That's correct for QGIS server. I'll first wait on sbrunner for GEOServer. Thanks